tests/phpunit/src/CommandTestBase.php

```diff
@@ -74,14 +69,16 @@ protected function setCommand(Command $command): void {
    *   stream.
    *
    * @throws \Exception
+   * @throws \Psr\Cache\InvalidArgumentException
    */
   protected function executeCommand(array $args = [], array $inputs = []): void {
     $cwd = $this->projectFixtureDir;
     chdir($cwd);
-    $tester = $this->getCommandTester();
-    $tester->setInputs($inputs);
-    $command_name = $this->command->getName();
-    $args = array_merge(['command' => $command_name], $args);
+
+    $input = new ArrayInput($args);
```

Comment:
The idea of this refactor is to delay creation of $input until just before the command is executed so that our container and our executed command both use exactly the same input object.